src/examples/FlyoutExamplePage.tsx

```diff
@@ -0,0 +1,322 @@
+'use strict';
+import {Text, View, TouchableHighlight} from 'react-native';
+import React, {useState, useRef} from 'react';
+import {Example} from '../components/Example';
+import {Page} from '../components/Page';
+import {Flyout} from 'react-native-windows';
+
+export const FlyoutExamplePage: React.FunctionComponent<{}> = () => {
+  const [showFlyout1, setShowFlyout1] = useState(false);
+  const [showFlyout2, setShowFlyout2] = useState(false);
+  const [showFlyout3, setShowFlyout3] = useState(false);
+  const [showFlyout4, setShowFlyout4] = useState(false);
+
+  const myRef = useRef();
+
+  const example1jsx = `<TouchableHighlight
+  onPress={() => {
+    setShowFlyout1(true);
+  }}>
+  <Text>Open Popup</Text>
+</TouchableHighlight>
+<Flyout
+  isOpen={showFlyout1}
+  onDismiss={() => {
+    setShowFlyout1(false);
+  }}>
+  <View>
+    <Text>This is a flyout.</Text>
+    <TouchableHighlight
+      onPress={() => {
+        setShowFlyout1(false);
+      }}>
+      <Text>Close Flyout</Text>
+    </TouchableHighlight>
+  </View>
+</Flyout>`;
+  const example2jsx = `<TouchableHighlight
+  onPress={() => {
+    setShowFlyout2(true);
+  }}>
+  <Text>Open Popup</Text>
```

Comment:
>Popup [](start = 13, length = 5)

I suggest renaming this to "Open Flyout" - as Popup is an implementation detail (think of Flyout as a 1st class popup-like component, rather than a component that has a Popup)

Reply:
Updated.